gateway/modules/crud/mgo/mongo.go

```diff
@@ -70,7 +73,16 @@ func (m *Mongo) connect() error {
 	ctx, cancel := context.WithTimeout(context.Background(), timeOut)
 	defer cancel()
 
-	client, err := mongo.NewClient(options.Client().ApplyURI(m.connection))
+	opts := options.Client().ApplyURI(m.connection)
+	opts = opts.SetMaxPoolSize((uint64)(m.driverConf.MaxConn))
```

Comment:
Set default values for each one of the driver configs first. However, don't overwrite the driver conf object, make new variables